docs/collecting-data/collecting-from-own-applications/google-amp-tracker/version-1/index.md

```diff
@@ -301,7 +296,7 @@ v2 of the tracker brings with it significant improvements in our ability to mode
 
 All events sent via this tracker will have:
 
-- `v_tracker` set to `amp-1.0.0`
+- `v_tracker` set to `amp-1.1.0`
```

Comment:
Maybe we should change this to use the variable from the versions file? (and the reference in the title also perhaps)

Reply:
Yeap tried to do those with just `Frontmatter` substitutions but did not seem to work. Maybe @mscwilson can help me out.